UPGRADING.md

```diff
@@ -0,0 +1,8 @@
+Upgrading SunspotMongo
+======================
+
+### Upgrading to >= 1.2.0
+
+The Rake task `sunspot:mongoid:reindex` has been renamed to `sunspot:reindex` and now overrides the original Rake task from Sunspot.
+
+See [#9](https://github.com/ruby-grape/grape/pull/9) for more information.
```

Comment:
Ouch copy-paste!

In this file you wee what I mean about version bump: we need some kind of reference in the project's history from which this upgrade note is relevant. Like most projects, Garpe is using version numbers is reference.

Now the point is what number the new version will be.
Do you follow [semantic versioning](http://semver.org/)?

```
Given a version number MAJOR.MINOR.PATCH, increment the:
MAJOR version when you make incompatible API changes
```
The renaming of the rake task is indeed an incompatible change, but I think that moving to 2.0.0 would lead the gem's users to think that there is a considerable difference, which is not the case.... What are your thoughts about this?